lib/cli.ml

```diff
@@ -45,7 +45,7 @@ let cmd_new =
 let cmd_push =
   Command.basic ~summary:"Push the current branch to origin"
     (let%map_open.Command force =
-       flag "f" ~aliases:[ "--force" ] no_arg
+       flag "f" ~aliases:[ "--force-with-lease" ] no_arg
```

Comment:
I guess this lacks a `--force` flag that would use git's `--force`.

Reply:
I actually do want to force push quite often (and not --force-with-lease). E.g. when amending the last commit

Reply:
Amending the last commit is the perfect use case for --force-with-lease…

Reply:
I got it confused with fast-word 🥴 

Okay, after reading the `git` manual I now agree that `--force-with-lease` is a better default indeed.

Would you be able to update the CHANGELOG.md as well? Happy to merge after that 🙂